data/announcements/scheduled.yaml

```diff
@@ -174,3 +174,13 @@ announcements:
     k8s.gcr.io image registry is gradually being redirected to registry.k8s.io (since Monday March 20th).<br>
     All images available in k8s.gcr.io are available at registry.k8s.io.</br>
     Please read our [announcement](/blog/2023/03/10/image-registry-redirect/) for more details.
+
+- name: Freezing the legacy package repositories
+  startTime: 2023-09-01T00:00:00
+  endTime: 2023-11-04T00:00:00 # Until KubeCon NA 2023
+  style: "background: #c70202"
+  title: The legacy package repositories will be frozen starting with September 13, 2023
```

Comment:
- On or around 13th of September, the banner should change to a “have been frozen” message.
- I would drop the banner after September.

Reply:
Both comments are addressed, please check when you have some time